### ISSUE TITLE
Add space around alt text in epub css

### ISSUE THREAD

**Issue by nelliemckesson** [macmillanpublishers/bookmaker_assets#30]
Alt text should have some space above and below the first and last paragraphs in a block. E.g. (from 9781627794671 in egalleymaker):

![screenshot 2016-09-27 17 18 53](https://cloud.githubusercontent.com/assets/812739/18892172/80f81ac4-84d6-11e6-9c23-199d6f3f6176.png)
